### PR TITLE
Update link to analytics documentation

### PR DESCRIPTION
**Pull request by pierlon**
## Summary

Amends the link to the analytics plugin documentation.

## Checklist

- [ ] My pull request is addressing an open issue (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

**Review by amedina — COMMENTED** [review]
Ship it!

File: assets/src/settings-page/analytics.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ export function Analytics() {
 							__( 'Please see AMP project\'s <a href="%1$s" target="_blank">documentation</a> for %2$s as well as the <a href="%3$s" target="_blank">plugin\'s analytics documentation</a>. Each analytics configuration supplied below must take the form of a JSON object beginning with a %4$s and ending with a %5$s. Do not include any HTML tags like %6$s or %7$s. For the type field, supply one of the <a href="%8$s" target="_blank">available analytics vendors</a> or leave it blank for in-house analytics. For Google Analytics specifically, the type should be %9$s.', 'amp' ),
 							__( 'https://amp.dev/documentation/components/amp-analytics/', 'amp' ),
 							'<code>amp-analytics</code>',
-							__( 'https://amp-wp.org/documentation/playbooks/analytics/', 'amp' ),
+							__( 'https://amp-wp.org/documentation/getting-started/analytics/', 'amp' ),
```
Comment (westonruter) [inline assets/src/settings-page/analytics.js:215]: Humm. When I navigate to https://amp-wp.org/documentation/getting-started/analytics/ I am redirected to https://amp-wp.org/documentation/playbooks/analytics/

**Review by westonruter — CHANGES_REQUESTED** [review]
How was the need for this change determined?

File: assets/src/settings-page/analytics.js (hunk 1 above)
Comment (westonruter) [reply assets/src/settings-page/analytics.js:215]:
```
$ curl -is https://amp-wp.org/documentation/getting-started/analytics/ | grep location
HTTP/2 301
...
location: https://amp-wp.org/documentation/playbooks/analytics/
```

Comment (westonruter) [reply assets/src/settings-page/analytics.js:215]: I see. This will be the new documentation URL on the refreshed site.

**Review by westonruter — APPROVED** [review]

**Pull request merged**